catalogs/catalog-jdbc-doris/src/main/java/org/apache/gravitino/catalog/doris/operation/DorisTableOperations.java

```diff
@@ -164,6 +166,37 @@ protected String generateCreateTableSql(
     return result;
   }
 
+  private Map<String, String> appendNecessaryProperties(Map<String, String> properties) {
+    Map<String, String> resultMap;
+    if (properties == null) {
+      resultMap = new HashMap<>();
+    } else {
+      resultMap = new HashMap<>(properties);
+    }
+
+    // If the backend server is less than 3, we need to set the property 'replication_allocation'
+    // to 1 explicitly.
+    if (!properties.containsKey("replication_num")) {
```

Comment:
why don't you use `tablePropertiesmetadata`?

Reply:
Do you mean we can put property `replication_num` into `tablePropertiesmetadata`?

Reply:
No. Add a new class such as `DorisTablePropertiesMetadata` to manage the property